google-cloud-spanner-executor/pom.xml

```diff
@@ -30,6 +30,11 @@
       <groupId>io.grpc</groupId>
       <artifactId>grpc-api</artifactId>
     </dependency>
+    <dependency>
+      <groupId>com.google.cloud</groupId>
+      <artifactId>grpc-gcp</artifactId>
+      <version>1.6.1</version>
```

Comment:
don't we centralize the versions or create separate dependencyManagement?
on upgrades, it may be hard to find and update all such version tags